config/govuk_pages.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Should this file be checked in? It's auto-generated when the differ runs, so it might not be?